src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -62,14 +81,12 @@
 								:title="item.label"
 								@click="createAndJoinConversation(item)">
 								<template #icon>
-									<ConversationIcon :item="iconData(item)"
-										:disable-menu="true" />
+									<ConversationIcon :item="iconData(item)" :disable-menu="true" />
```

Comment:
I'd revert this commit: https://github.com/nextcloud/spreed/pull/9688/commits/41dff5ce63f5dff91a8e19c80de62d4e00560c56
What worries me, as it touches a lot of lines of code, and it will be difficult to backport something later (because this PR doesn't go to NC27, as it's a new feature for NC28)
If you want, we could made a specific PR for it later